sklearn/metrics/classification.py

```diff
@@ -1187,9 +1163,7 @@ def precision_score(y_true, y_pred, labels=None, pos_label=1,
            parameter *labels* improved for multiclass problem.
 
     pos_label : str or int, 1 by default
-        The class to report if ``average='binary'``. Until version 0.18 it is
-        necessary to set ``pos_label=None`` if seeking to use another averaging
-        method over binary targets.
+        The class to report if ``average='binary'`` and the data is binary.
```

Comment:
Add a note here maybe, or in the docstring saying how to select a single class in the multi-class setting?